setup.py

```diff
@@ -67,9 +67,8 @@ def generate_certificate_request(kms_key: str, subject: str):
     pubkey_der = response['PublicKey']
 
     # Create temporary private key
-    private_key = rsa.generate_private_key(
-        public_exponent=65537,
-        key_size=2048,
+    private_key = ec.generate_private_key(
+        ec.SECP384R1()
```

Comment:
The should be whatever maps to PRIME256V1

Reply:
Since you are using NIST the name could be secp256r1

Reply:
Updated to use `ec.SECP256R1()`